### ISSUE TITLE
PascalCase constant is classified as a component

### ISSUE THREAD

**Issue by githorse** [ArnaudBarre/eslint-plugin-react-refresh#71]
I think perhaps I'm seeing the converse of https://github.com/ArnaudBarre/eslint-plugin-react-refresh/issues/26: I use PascalCase for my constants, and the plugin considers erroneously them to be components. An example that will trigger the rule:

```js
export const SomeConstant = 42 // ✗  [eslint] Fast refresh only works when a file only exports components ... 
export function someUtility() { return SomeConstant }
```

If I change that the capitalization to screaming snake, it passes:
```js
export const SOME_CONSTANT = 42 // ✓
export function someUtility() { return SOME_CONSTANT }
```

OK, I guess PascalCase is not the preferred convention here. But is there any global setting I'm missing that could easily prevent such constants from being flagged? (Using `eslint-disable-next-line` is more distracting than it's worth; if I have to do that I'll probably just switch to screaming snake, but that's not my preference.)

**Comment by ArnaudBarre** [comment]
Are you using a recent version? I don't have issue a warning with default options for this code:
```
export constant SomeConstant = 42 // ✗  [eslint] Fast refresh only works when a file only exports components ... 
export function someUtility() { return SomeConstant }
```

**Comment by RazerM** [comment]
I had a file like this and the `const RETRIES` was being flagged, but renaming `createRoutes` to fit the component naming convention (e.g. to `AppRoutes`) fixed it.

```js
const RETRIES = 3

export default function createRoutes() {
    return <App />
}
```

**Comment by githorse** [comment]
Ok, I'm not able to reproduce this at the moment either ... I may have been conflating it with [a slightly different issue]( https://github.com/ArnaudBarre/eslint-plugin-react-refresh/issues/75). Closing this one (at least until I can reproduce).
